docs/UsersGuide/source/BackgroundInfo/CCPPUpdates.rst

```diff
@@ -0,0 +1,95 @@
+:orphan:
+
+.. _CCPPUpdates:
+
+================================================
+CCPP Updates for the SRW App v2.2.0 Release
+================================================
+
+Here is what's new in CCPP Physics for the UFS SRW v2.2.0 public release. 
+
+General Updates
+=================
+
+* Added RAP suite (``FV3_RAP``) as a new supported suite (documentation `here <https://dtcenter.ucar.edu/GMTB/UFS_SRW_App_v2.2.0/sci_doc/rap_suite_page.html>`__)
+* Added the Community Land Model (CLM) Lake model in the HRRR suite (``FV3_HRRR``)
+
+Thompson Microphysics Scheme
+==============================
+
+* Reduced ice generation supersaturation requirement from 0.25 to 0.15 to generate more ice at the upper levels and reduce the outgoing longwave radiation bias
+* Divided cloud number concentration into two parts (over land and others). Reduced number concentration over ocean to a smaller number (50/L) from its previous default (100/L). Both changes were made to reduce excessive surface downward shortwave radiative flux off coastal regions including the Southeast Pacific
+* Implemented small fixes to the minimum size of snow and collision constants
+
+.. note:: 
+   
+   The above improvements were tested with the non-aerosol option, so results with the aerosol-aware Thompson (used in the SRW App) may vary.
+
+
+NoahMP Land Surface Model
+===========================
+
+* Option for using the unified frozen precipitation fraction in NoahMP.
+* Diagnostic 2-meter temperature and humidity now based on vegetation and bare-ground tiles (new namelist option ``iopt_diag``)
+* Bug fixes for GFS-based thermal roughness length scheme
+* New soil color dataset introduced to improve soil albedo to reduce the large warm bias found in the Sahel desert
+* Wet leaf contribution factor is included
+* Leaf-area index now depends on momentum roughness length
+
+
+RUC Land Surface Model
+========================
+
+* Initialization of land and ice emissivity and albedo with consideration of partial snow cover
+* Initialization of water vapor mixing ratio over land ice
+* Initialization of fractions of soil and vegetation types in a grid cell
+* Changes in the computation of a flag for sea ice: set to true only if ``flag_cice=.false`` (atmosphere uncoupled from the sea ice model).
+* Separate variables for sea ice, for example: ``snowfallac`` is replaced with ``snowfallac_ice``
+* Solar angle dependence of albedo for snow-free land
+* Stochastic physics perturbations (SPP) introduced for emissivity, albedo and vegetation fraction
```

Comment:
@JeffBeck-NOAA This item came from Man Zhang; I reached out to her to ask if she knows what changes this is in reference to.

Reply:
Unfortunately it looks like there was some miscommunication when we were developing the release notes...Man added some things that were included in previous releases because of documentation we got from the RUC-LSM developers. So the SPP bullet point should be removed for complete accuracy. There may be other duplicates from previous releases as well, do you think it is worth going back through the list completely?

Reply:
Ok, I removed the SPP bullet. As to whether to go back through the list completely, I'd say no. The CCPPUpdates file is already set in the release as-is; nothing we do here changes that. If there is a desire to go through and make corrections to the release docs, we could then bring those changes over to `develop`, too, but I'd prefer to make it all a different PR. This PR needs to be merged with minimal delay because it's already been 2.5 months since the previous release (not trying to lay blame--I let things sit for quite a bit, too, while working on the Land DA release). I think the benefit of merging soon outweighs the limited downside of a few bullet points that are inaccurate only in terms of timeline (not in terms of content present in the SRW).

Reply:
Thanks, @gspetro-NOAA, @mkavulich!